zerver/lib/events.py

```diff
@@ -827,3 +877,13 @@ def post_process_state(ret: Dict[str, Any]) -> None:
             d.pop('is_active')
 
         del ret['raw_users']
+
+    if 'raw_recent_conversations' in ret:
+        # Reformat recent_conversations to be a list of dictionaries, rather than a dict.
+        ret['recent_conversations'] = sorted([
+            dict(
+                recipient_id=recipient_id,
```

Comment:
We can skip including `recipient_id` in what we send down to clients; see https://github.com/zulip/zulip-mobile/issues/3133#issuecomment-475115028